apis/v1alpha1/query.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+type QueryVariable struct {
```

Comment:
I see types in this file and variable.go are not used, are they added by accident or planned for future use?

Reply:
They are for future use, but I didn't mean to include them in this PR. I removed them.